server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -66,6 +68,14 @@ public void upgradeMetadata(ServerContext context) {
     deleteExternalCompactions(context);
   }
 
+  private void checkForRemovedProperties(ServerContext context) {
+    String lastLocationModeProperty = "tserver.last.location.mode";
```

Comment:
Not sure if this would work, wondering if we can call `ConfigCheckUtil.validate()` here and it would warn about any props that are not valid in the new version of the code.  Saw other part of the code were calling this.

Reply:
Implemented checks using `ConfigCheckUtil.validate` in 4d279e7.